swan-lake/other/tooling-guide/intellij-idea-plugin/set-up-intellij-idea/use-intellij-plugin-features 2.md

```diff
@@ -149,7 +149,7 @@ You expand/collapse the following Ballerina code segments using the icons in the
 
 ## Go to definition
 
-This option allows you to view the definition of a selected variable, function, an object etc., within the same file, in a separate file, in the same module, or in a file of a different module, of the same project or of the [Standard Library](/learn/api-docs/ballerina/).
+This option allows you to view the definition of a selected variable, function, an object etc., within the same file, in a separate file, in the same module, or in a file of a different module, of the same project or of the [Ballerina library](/learn/api-docs/ballerina/).
```

Comment:
Is this a valid link. anyway not correct to point API docs of `ballerina` org only as Ballerina library should contain API docs of ballerina and ballerianx orgs.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/files.